Acala/acala-evm-starter/docker-compose.yml

```diff
@@ -35,6 +35,10 @@ services:
       - ${SUB_COMMAND} # set SUB_COMMAND env variable to "test" to run tests
       - -f=/app
       - --db-schema=app
+      - --workers=4
+      - --batch-size=30
```

Comment:
It would be good to have comments explaining what these do and how/why users would want to change these

Does it make sense for us to change the default batch size to 30 in the sdk?